style.css

```diff
@@ -0,0 +1,77 @@
+/* @media all and (max-width: 500px) { */
+* {
+  font-family: sans-serif;
+  padding-left: 1%;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+}
```

Comment:
As requested by the previous reviewer, the blank empty space to the right side of the page is still present, :cry:  this could be as a result of setting a fixed width of 327px in almost all your section as seen in lines 8, 32, 49, 68 etc. :1234: Kindly consider removing them. If you would give a width, consider using percentage (%) or vw units which are more flexible and responsive :+1: 

Your design

![Screenshot from 2022-12-02 08-24-25](https://user-images.githubusercontent.com/65233860/205250056-66b936c2-1711-4fe9-b0d4-945853bcce09.png)

The figma design

![Screenshot from 2022-12-02 08-24-48](https://user-images.githubusercontent.com/65233860/205250074-ab4afe0f-d70d-4d7d-bdb7-46f936c351e9.png)